docs/command_line_reference.rst

```diff
@@ -975,3 +975,19 @@ Examples, assuming that two clusters have been specified with ``--target-hosts``
 
 .. WARNING::
    If you use ``client-options`` you must specify options for **every** cluster name defined with ``target-hosts``. Rally will raise an error if there is a mismatch.
+
+Command line parameters accepting comma-separated values (csv)
```

Comment:
I must be weird, but seeing "csv" immediately makes me think about CSV files that can be separated by newlines, and we're not supporting newlines.

```suggestion
Command line parameters accepting comma-separated values
```

Reply:
Fixed in 837bd15ba7890f1ba055324855f08d0014e20e98